### PR TITLE
Add missing fields of powershell

### PR DESCRIPTION
**Pull request by xian123**
## Description
There is a gap between public SDK and powershell of GetIntegrationRuntimeStatus result:
SDK: https://docs.microsoft.com/en-us/dotnet/api/microsoft.azure.management.datafactory.models.selfhostedintegrationruntimestatus?view=azure-dotnet
Powershell: https://github.com/Azure/azure-powershell/blob/master/src/DataFactory/DataFactoryV2/Models/PSSelfHostedIntegrationRuntimeStatus.cs
We need to add the missing fields of powershll.

## Checklist

* [x] I have read the [_Submitting Changes_](../blob/master/CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](../blob/master/CONTRIBUTING.md)
* [x] The title of the PR is clear and informative
* [x]The appropriate `ChangeLog.md` file(s) has been updated:
    - For any service, the `ChangeLog.md` file can be found at `src/{{SERVICE}}/{{SERVICE}}/ChangeLog.md`
    - A snippet outlining the change(s) made in the PR should be written under the `## Upcoming Release` header -- no new version header should be added
* [x] The PR does not introduce [breaking changes](../blob/master/documentation/breaking-changes/breaking-changes-definition.md)
* [x] If applicable, the changes made in the PR have proper test coverage
- [ ] For public API changes to cmdlets:
    - [ ] a cmdlet design review was approved for the changes in [this repository](https://github.com/Azure/azure-powershell-cmdlet-review-pr) (_Microsoft internal only_)
    - [ ] the markdown help files have been regenerated using the commands listed [here](../blob/master/documentation/development-docs/help-generation.md#updating-all-markdown-files-in-a-module)

### CODE REVIEW

**Comment by msJinLei** [comment]
@xian123 
- Change the changelog.md
- update or add at least one related test case and update recorded json

**Comment by adxsdkps** [comment]
Can one of the admins verify this patch?

**Comment by xian123** [comment]
Update the ChangeLog.md, test case script and help file.

File: ChangeLog.md [hunk 1]
```diff
@@ -57,6 +57,9 @@
 * Fix the issue that Set-AzStorageContainerAcl can clean up the stored Access Policy
 	- Set-AzStorageContainerAcl
 
+#### Az.DataFactory
```
Comment (msJinLei) [inline ChangeLog.md]: Update the changelog.md in your model's path

Comment (xian123) [reply ChangeLog.md]: Update the code, please have a review again.

**Comment by xian123** [comment]
@zhangyd2015 @msJinLei Please have a review.

File: src/DataFactory/DataFactoryV2/Changelog.md [hunk 2]
```diff
@@ -22,6 +22,7 @@
 ## Version 1.5.1
 * Update ADF .Net SDK version to 4.5.0
 * Update references in .psd1 to use relative path
+* Add AutoUpdateETA, LatestVersion, PushedVersion, TaskQueueId and VersionStatus properties
```
[the remark this answers is not in the document]
Comment (xian123) [reply src/DataFactory/DataFactoryV2/Changelog.md]: Updated.

File: src/DataFactory/DataFactoryV2/Changelog.md [hunk 3]
```diff
@@ -18,6 +18,7 @@
         - Additional information about change #1
 -->
 ## Upcoming Release
+* Add AutoUpdateETA, LatestVersion, PushedVersion, TaskQueueId and VersionStatus properties
```
Comment (zhangyd2015) [inline src/DataFactory/DataFactoryV2/Changelog.md]: You will need to explain the properties belong to which cmdlet.

File: src/DataFactory/DataFactoryV2.Test/ScenarioTests/IntegrationRuntimeTests.ps1 [hunk 4]
```diff
@@ -64,6 +64,12 @@ function Test-SelfHosted-IntegrationRuntime
             -Force
         Assert-AreEqual $result.Description $description
 
+        $status = Get-AzDataFactoryV2IntegrationRuntime -ResourceGroupName $rgname `
+            -DataFactoryName $dfname `
+            -Name $irname `
+            -Status
+        Assert-NotNull $status.LatestVersion
+
```
Comment (zhangyd2015) [inline src/DataFactory/DataFactoryV2.Test/ScenarioTests/IntegrationRuntimeTests.ps1]: If you change the scenario tests, you will need to re-record the flow, following [recordingrunning-tests](https://github.com/Azure/azure-powershell/blob/master/documentation/development-docs/azure-powershell-developer-guide.md#recordingrunning-tests)

Comment (xian123) [reply src/DataFactory/DataFactoryV2.Test/ScenarioTests/IntegrationRuntimeTests.ps1]: Updated the codes.

File: src/DataFactory/DataFactoryV2.Test/SessionRecords/Microsoft.Azure.Commands.DataFactoryV2.Test.IntegrationRuntimeTests/TestSelfHostedIntegrationRuntime.json [hunk 5]
```diff
@@ -1,106 +1,99 @@
 {
   "Entries": [
     {
-      "RequestUri": "/subscriptions/c39dce18-cead-4065-8fb1-3af7683a5038/resourcegroups/ps2452?api-version=2016-09-01",
-      "EncodedRequestUri": "L3N1YnNjcmlwdGlvbnMvYzM5ZGNlMTgtY2VhZC00MDY1LThmYjEtM2FmNzY4M2E1MDM4L3Jlc291cmNlZ3JvdXBzL3BzMjQ1Mj9hcGktdmVyc2lvbj0yMDE2LTA5LTAx",
+      "RequestUri": "/subscriptions/b371d9e7-d3c2-4b1a-83ec-84e1f50c2222/resourcegroups/ps5033?api-version=2016-09-01",
```
Comment (msJinLei) [inline src/DataFactory/DataFactoryV2.Test/SessionRecords/Microsoft.Azure.Commands.DataFactoryV2.Test.IntegrationRuntimeTests/TestSelfHostedIntegrationRuntime.json]: In the test case, api version is 2018-06-01

/subscriptions/b371d9e7-d3c2-4b1a-83ec-84e1f50c2379/resourceGroups/ps5033/providers/Microsoft.DataFactory/factories/ps1068?api-version=2018-06-01

File: tools/Tools.Common/SerializedCmdlets/Microsoft.Azure.PowerShell.Cmdlets.DataFactoryV2.dll.json [hunk 6]
```diff
@@ -24256,6 +24256,7 @@
               "ServiceUrls": "System.Collections.Generic.IList`1[System.String]",
```
Comment (msJinLei) [inline tools/Tools.Common/SerializedCmdlets/Microsoft.Azure.PowerShell.Cmdlets.DataFactoryV2.dll.json]: Please revert the change to this file
